bots/tests-invoke

```diff
@@ -373,8 +373,13 @@ class PullTask(object):
 
 
         env = os.environ.copy()
-        if "TEST_DATA" in env:
-            del env["TEST_DATA"]
+        # we can only use the cache for master; branches still use the old
+        # test/images/ symlink farm which we must not destroy
+        if opts.rebase:
+            if "TEST_DATA" in env:
+                del env["TEST_DATA"]
+        else:
+            env["TEST_DATA"] = BOTS
```

Comment:
That was the value [before](https://github.com/cockpit-project/cockpit/commit/3a098012aa#diff-e8a6006ae7f521560b0a4a77543d50ffL334), I assumed it was due to some path rewrite between host and container. But quite obviously it didn't work, in the [log](https://fedorapeople.org/groups/cockpit/logs/pull-7236-20170712-215050-4ab5d425-verify-ubuntu-1604/log.html) there are several image downloads again. I'll try again with just keeping it.